manpages/brew-cask.1

```diff
@@ -162,6 +162,10 @@ Target location for Preference Panes\. The default value is \fB~/Library/Prefere
 Target location for QuickLook Plugins\. The default value is \fB~/Library/QuickLook\fR\.
 .
 .TP
+\fB\-\-dictionarydir=<path>\fR
```

Comment:
Sorry if I wasn't clear; you still need to commit `brew-cask.1.md` and `brew-cask.1`.